tests/runner/test_sequential_runner.py

```diff
@@ -278,3 +279,77 @@ def test_confirms(self, mocker, pipeline, is_async):
         catalog = DataCatalog(data_sets={"ds1": fake_dataset_instance})
         SequentialRunner(is_async=is_async).run(pipeline, catalog)
         fake_dataset_instance.confirm.assert_called_once_with()
+
+
+@pytest.fixture
+def resume_scenario_pipeline():
+    return Pipeline(
```

Comment:
```suggestion
    return pipeline(
```
very minor, but ideally all tests would use `pipeline` rather than `Pipeline` now (there's lots of other instances of this which seem to have been missed outside your code here - feel free to change them all).

Reply:
Sure, I made this change for all instances of Pipeline in the file in [e4a6525](https://github.com/kedro-org/kedro/pull/1795/commits/e4a65257d878d8aa9776923c2c6d58ad25417610).

Where this created naming conflicts with variables / parameters, I renamed these `test_pipeline`.